### PR TITLE
Mantas 02 registration

### PR DESCRIPTION
**Pull request by ManNavic**

### CODE REVIEW

**Comment by Shylan21** [comment]
Nice coding, looks good :)

File: src/context/auth.js [hunk 1]
```diff
@@ -19,11 +19,12 @@ const AuthProvider = ({ children }) => {
 
         if (storedToken && !token) {
             setToken(storedToken)
+			console.log(storedToken)
```
Comment (Auenc) [inline src/context/auth.js:22]: Let's remove this console log

File: src/pages/register/index.js [hunk 2]
```diff
@@ -1,54 +1,105 @@
...
+      } else {
+        setPasswordError('');
+      }
+    }
+    setFormData({ ...formData, [name]: value });
+  };
+
+  const onSubmit = async (e) => {
+    e.preventDefault();
+    console.log(formData.email, formData.password);
```
Comment (Auenc) [inline src/pages/register/index.js:53]: Let's remove this console log

**Review by Auenc — CHANGES_REQUESTED** [review]

**Review by Auenc — CHANGES_REQUESTED** [review]
Minor changes mentioned in the comments

**Review by Auenc — DISMISSED** [review]

**Review by Auenc — DISMISSED** [review]

**Comment by procrastian** [comment]
All good!

**Review by JayVith — DISMISSED** [review]
All changes LGTM!

**Review by dearshrewdwit — CHANGES_REQUESTED** [review]
Only one change - double check it's reasonable.

❗ In future, please don't include in a PR style changes like double quotes to single quotes.
Otherwise everyone's PRs will include these changes going back and from different quotes.

The action here -> 
1. Either agree as a cohort which style to follow, and then enforce it
2. Don't change them

**Comment by procrastian** [comment]
LGTM

**Review by dearshrewdwit — APPROVED** [review]

**Review by procrastian — APPROVED** [review]
LGTM!

**Pull request merged**